creativecloud/blocks/firefly-model-showcase/firefly-model-showcase.css

```diff
@@ -275,6 +275,10 @@
     --content-side-padding: calc((100% - 1440px + 16px) / 2);
     --content-padding: 0 0 var(--content-bottom-padding) var(--content-side-padding);
   }
+  
+  html[dir="rtl"] .firefly-model-showcase-content {
```

Comment:
@suhjainadobe - I thought @robert-bogos was supposed to handle this. https://github.com/adobecom/cc/pull/828